core/crossmodel/interface.go

```diff
@@ -101,8 +101,11 @@ type ApplicationOfferFilter struct {
 	// Endpoint contains an endpoint filter criteria.
 	Endpoints []EndpointFilterTerm
 
-	// AllowedUsers are the users allowed to consume the application.
+	// AllowedUsers are the users allowed to consume the offer.
 	AllowedUsers []string
+
+	// ConnectedUsers are the users currently related to the offer.
+	ConnectedUsers []string
```

Comment:
The doc strings here seem to match with my understanding, but you clearly have a test case that --user goes to ConnectedUsers and --consumer goes to AllowedUsers.

Reply:
--user does go to connected user (ie the offers where that --user has at least one relation to the offer)
--consumer does go to allowed user (ie the offers where the specified user is allowed to be a consumer)